### DISCUSSION TITLE
关于

### DISCUSSION THREAD

**Discussion by giscus** [darkyzhou/blog-comments#4] [General]
# 关于

 我是谁
我叫 Darkyzhou，以前的 ID 是 Darkyoooooo，也可以简写成 Darky。朋友们因为一个奇怪的食用油广告，总是称我为“达克油”，但其实我不会制油。我生于 2001 年，在粤北的一座不算安静的小山城里。
我从哪来
我成长于一座粤北小城，9 岁那年家里有了第一台电脑。2012 年的春天，我接触...

https://darkyzhou.net/about

**Comment by ganxiaozhe** [comment]
Hello，在检索 Sveltekit Blog 静态站点生成器时从 Github 偶然看到你的 blog。
似乎好多人都是受到 Minecraft 的影响，从此打开了编程的大门，[也包括我。](https://gxzv.com/about/cv/)
喜欢你文字带给我的感觉，但因为此时还有一些事情待完成，暂时无法一一细读。

总之很高兴认识你。
祝好，我还会回来的 :)

**Comment by xhj967** [comment]
你好，很高兴认识你，我给你发了一个邮件，请教一下问题xhj967@163.com

**Comment by yaowencurry** [comment]
这个博客做的太厉害了，点赞👍

**Comment by Master-Hash** [comment]
从 https://github.com/rust-lang/rust-analyzer/issues/18578 偶然发现了这里。

我也主要搞前端（很巧，我也最近用 Waku 重写了自己网站），也打一点包（不过不是 Linux 发行版，是 MSYS2），大概同行的世界就是这么小。

很高兴认识你！
